clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/input/recordreader/vson/VeniceVsonFileIterator.java

```diff
@@ -16,7 +16,7 @@
 public class VeniceVsonFileIterator implements VeniceRecordIterator {
   private static final Logger LOGGER = LogManager.getLogger(VeniceVsonRecordReader.class);
 
-  private SequenceFile.Reader fileReader;
+  private final SequenceFile.Reader fileReader;
```

Comment:
Maybe rename `fileReader` to `sequencefileReader`?

Reply:
I don't think this is needed. It seems redundant info as:
* The file is a Vson file
* All Vson files are sequence files
* The class name contains Vson in the name already